sdk/src/Core/Amazon.Runtime/Credentials/_bcl+netstandard/SSOImmutableCredentials.cs

```diff
@@ -39,11 +39,24 @@ public class SSOImmutableCredentials : ImmutableCredentials
         /// <param name="expiration">The expiration time for the credentials.</param>
         public SSOImmutableCredentials(
             string awsAccessKeyId, string awsSecretAccessKey,
-            string token, DateTime expiration)
-            : base(awsAccessKeyId, awsSecretAccessKey, token)
+            string token, DateTime expiration) : this(awsAccessKeyId, awsSecretAccessKey, token, expiration, null)
+        {
+        }
+
+        /// <summary>
+        /// Constructs an instance with supplied keys, token, expiration, and account id
+        /// </summary>
+        /// <param name="awsAccessKeyId"></param>
+        /// <param name="awsSecretAccessKey"></param>
+        /// <param name="token"></param>
+        /// <param name="expiration"></param>
+        /// <param name="accountId"></param>
+        /// <exception cref="ArgumentNullException"></exception>
+        public SSOImmutableCredentials(string awsAccessKeyId, string awsSecretAccessKey, string token, DateTime expiration, string accountId) : base(awsAccessKeyId, awsSecretAccessKey, token, accountId)
         {
             if (string.IsNullOrEmpty(token)) throw new ArgumentNullException(nameof(token));
             Expiration = expiration;
+            AccountId = accountId;
```

Comment:
You are passing `accountId` to the base constructor so setting this again here does not seem necessary.